.github/actions/install-dependencies/action.yml

```diff
@@ -3,7 +3,7 @@ description: 'Set up node and install dependencies'
 runs:
   using: 'composite'
   steps:
-    - uses: actions/setup-node@v3
+    - uses: actions/setup-node@v4
```

Comment:
I noticed Dependabot missed this one. I can't recall if there was an issue with nested actions before, but did you want a PR to add the extra job?

Reply:
Is that something we can configure with Dependabot or a feature gap?

Reply:
I think I tried it before with a different `directory` value, but I don't remember if there was an issue https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot#enabling-dependabot-version-updates-for-actions.
I'll send something over, but don't hold this up on my part. I think you'll see it in the Insights>Dependancies tab after if it "works"

Reply:
Opened #532